src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section>
+      <div class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h4 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h4>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="stars">
+          <div class="stars__block stars-4">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+
+          <div class="stars__reviews">
+            Reviews: <span>5</span>
+          </div>
+        </div>
+
+        <div class="card__price price">
+          <p class="price-text">Price:</p>
```

Comment:
Incorrect, please delete class price from div, and rename classes for paragraph, for example card__price-text, card__price-cost